File: CTDK/081/081.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 0;
	float t = 1;
	int i = 0;
	while (i <= n)
	{
		t = t * (x + i);
		s = s + 1 / t;
		i++;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}

File: CTDK_Python/081/_081.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 1
m = 1
i = 1
while(i <= n):
    m = m * (x + i)
    s = s + 1 /m
    i = i + 1

print("Tong la: ", s)
